Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM gcr.io/world-fishing-827/github.com/globalfishingwatch/gfw-bash-pipeline:latest-python3.8
+
+# Install SDK. (needed for Python SDK)
+RUN pip install --no-cache-dir apache-beam[gcp]==2.56.0
+
+# Copy files from official SDK image, including script/dependencies.
+COPY --from=apache/beam_python3.8_sdk:2.56.0 /opt/apache/beam /opt/apache/beam
+
+# Perform any additional customizations if desired
+COPY ./requirements.txt ./
+RUN pip install -r requirements.txt
+
+# Temporary. TODO: Use a local test docker image with extra dependencies.
+COPY ./requirements/test.txt ./
+RUN pip install -r test.txt
+
+# Temporary. TODO: Use a local dev docker image with extra dependencies.
+COPY ./requirements/dev.txt ./
+RUN pip install -r dev.txt
```

Comment:
One thing we could do to solve this is to use multi-stage docker builds. You set up a base image that contains production dependencies, etc. Then you add a development stage that uses that base and add dev and test dependencies. Finally, you add a final productions stage that uses the base stage, and can set up entrypoints and so on. Then, you configure `compose.yaml` to use the development target when building. That means that locally (or in your CI) you can use the development stage and you have everything you need, but on production these dependencies are not installed because you use the development stage.

Reply:
For an example, take a look at [this dockerfile](https://github.com/ahoraquesinosven/api-registro-femicidios/blob/main/Dockerfile) and [this compose.yaml](https://github.com/ahoraquesinosven/api-registro-femicidios/blob/main/compose.yml). It's from an open source project I collaborate in outside GFW, and it's a nodejs API, but the approach is the same.

Reply:
Agree. Will create another ticket for this.

Reply:
To resolve at https://globalfishingwatch.atlassian.net/browse/PIPELINE-2053